### DISCUSSION TITLE
Disable single button press on Sonoff T0-1 gang (UK)

### DISCUSSION THREAD

**Discussion by mczac** [arendst/Tasmota#12183] [Q&A] [answered]
Hi there, I recently installed Sonoff T0 (UK) and was trying to find a way to disable single press completely to toggle the relay. I am comfortable with any other click events to toggle the relay. Read all the options, tried them but couldn't make it work. Is it feasible at all? What would the steps? Many thanks

**Comment by ascillato2** [comment, marked as answer]
You can use rules. Please, check for rules in http://tasmota.com

There is an example of exactly your request there.

**Reply by mczac** [reply]
if you are referring to this one, it doesn't work. I changed the GPIO0 Botton to Switch, then run all the commands below. Didn't work. 

single press: Does nothing (empty Delay commands)
hold 2 secs: Toggle Power1
```
SetOption11 0
Backlog SwitchTopic1 0; SwitchMode1 5; SetOption32 20  

Rule1
  ON Switch1#State=3 DO Power1 2 ENDON
  ON Switch1#State=2 DO Delay ENDON  
Rule1 1
```

**Reply by mczac** [reply]
all worked, thanks, my bad. had to change button to switch in module settings
